### PR TITLE
GRIP-17: Add scoping to templates

### PR DESCRIPTION
**Pull request by zyoung51**
- Update unittests to switch from findOne to find in on-core
- Depends on GRIP-17 PR in on-core
@RackHD/corecommitters

### CODE REVIEW

File: spec/lib/api/tasks-spec.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ describe('Http.Api.Tasks', function () {
 
         // mock waterline templates to return a template for bootstrap
         waterline = helper.injector.get('Services.Waterline');
-        waterline.templates.findOne = sinon.stub();
+        waterline.templates.find = sinon.stub();
```
Comment (benbp) [inline spec/lib/api/tasks-spec.js]: Ignoreable.

**Comment by benbp** [comment]
Only the test file was committed, missing the actual changes it seems.

**Comment by zyoung51** [comment]
Just the test files are affected as there are no changes to on-http to use the scope stuff yet.  For the tests, the observable behavior of get/render didn't change, but the tests specifically mock out findOne, which is no longer used.  I had to modify the test to specifically mock out find instead of findOne.

**Comment by benbp** [comment]
Ah ok, sorry I totally missed your "Depends on" comment.

**Comment by benbp** [comment]
:+1:

**Comment by anhou** [comment]
:+1:

**Pull request merged**